boards/samd11_bare/Cargo.toml

```diff
@@ -35,6 +35,9 @@ use_semihosting = []
 [profile.release]
 debug = true
 
+[profile.dev]
+opt-level = "s"
```

Comment:
any particular reason for this? Not opposed to adding it, though.

Reply:
Yeah, some examples (looking at you, i2c) won't compiled unoptimized because they take too much flash space.